pyroomacoustics/__init__.py

```diff
@@ -126,7 +126,14 @@
 
 from . import doa
 from . import adaptive
-from . import realtime
+from . import transform
+from . import transform as realtime  # to be deprecated
 from . import experimental
 from . import datasets
 from . import bss
+
+import warnings
```

Comment:
I know I asked for the deprecation warning. But is this the way to go ? The warning will be triggered everytime pyroomacoustics is loaded right ?

Reply:
According to here (https://docs.python.org/2/library/warnings.html). The warning will be ignored by default, as I added the "DeprecationWarning" label. Also it doesn't print out for me every time.

Reply:
this is a better link to the table: https://docs.python.org/2/library/warnings.html#warning-categories

Reply:
Cool. That sounds like a good way to do this.